layouts/partials/page-meta-links.html

```diff
@@ -29,19 +30,20 @@
     {{ $gh_repo_path = replaceRE . $ghs_rename $gh_repo_path -}}
   {{ end -}}
 
+  {{ $viewURL := printf "%s/tree/%s" $gh_repo $gh_repo_path -}}
   {{ $editURL := printf "%s/edit/%s" $gh_repo $gh_repo_path -}}
-  {{ $createURL := printf "%s/edit/%s" $gh_repo $gh_repo_path -}}
```

Comment:
I deleted this variable assignment because `$createURL` doesn't appear to be used anymore.